scripts/inventory/consul_io.py

```diff
@@ -57,25 +63,37 @@
 DNS docs for more info.
 
 which restricts the included nodes to those from the given datacenter
+This can also be set with the environmental variable CONSUL_DOMAIN
+
+'suffixes':
+
+boolean flag. By default, final JSON is built based on all available info in consul.
+Suffixes means that services groups will be added in addition to basic information. See servers_suffix for additional info
+There are cases when speed is preferable than having services groups
+False value will reduce script execution time drastically.
+This can also be set with the environmental variable CONSUL_SUFFIXES
```

Comment:
```suggestion
This can also be set with the environmental variable CONSUL_SUFFIXES.
```
Best to add that for all new env variable description lines.

Reply:
it's done for all env variables.